package.yaml

```diff
@@ -0,0 +1 @@
+# Toit Package.
```

Comment:
The MPL license isn't recognized by our "describe" tool.
-> add a license line?

The README file uses `toit-cert-roots` as title. That would give this package a preferred prefix/name of "toit_cert_roots".
However, in your example you use `certificate_roots`.
You could change the title of the README to "Certificate Roots" (which would make this package have a preferred prefix/name of `certificate_roots`. Alternatively you could add a `name` entry here.